### PR TITLE
Update numismatics form layout

### LINKED ISSUE #3623: Add links to manage numismatics people/monograms/citations etc in issue/coin form
When creating numismatics issues and coins, often it is necessary to create new people, monograms, citations, etc objects while entering the data into the form. Add links to manage the other numismatics object types within the context of the issue and coin forms.

### LINKED ISSUE #3624: Numismatics forms require too much scrolling
Numismatics staff reported that the old form could fit entirely on the screen without requiring any scrolling. The new system involves a lot of scrolling, which was reported as a negative user experience. The request was "please have less whitespace"

### LINKED ISSUE #3655: Should we turn on graphiql in production?
Question came up when preparing for pair interview. It might be helpful for people wanting to use our data more generally.

### LINKED ISSUE #3657: Remove Right Sidebar and Copyright fields for Numismatics Forms
Numismatics objects are always open access and copyright-free. Users expressed frustration with the amount of space the right sidebar takes up in the form.

### LINKED ISSUE #3681: Add icon to indicate when the monograms panel is collapsed/expanded
will add this in https://github.com/pulibrary/figgy/tree/3665-3657-3624-update-layout-form-numismatics

### PR DESCRIPTION
**Pull request by christinach**
* Move the monograms to the bottom of the page. Keep the Monograms panel collapsed when the page loads.
* Remove Rights and Notes panel from the Numismatics issue form
* Move the form control buttons (Save, Save and Duplicate, Cancel, Delete) to the bottom of the form
* Remove Progress-form sidebar from the Numismatics issue form.
* Add links with a button class to create new person, new place, new monogram. 
closes #3681 
closes #3655 
closes #3657 
closes #3624
closes #3623

### CODE REVIEW

File: app/views/base/_form_metadata.html.erb [hunk 1]
```diff
@@ -5,13 +5,17 @@
...
-    <div class="panel panel-default">
-      <% unless label.empty? %>
-        <div class="panel-heading"><%= label %></div>
-      <% else %>
-        <div class="panel-heading">Metadata</div>
-      <% end %>
-      <%= render partial: 'form_metadata_inner', locals: { fields: fields, f: f } %>
-    </div>
-  <% end %>
+    <% unless @change_set.model.is_a?(Numismatics::Issue) %>
```
Comment (hackmastera) [inline app/views/base/_form_metadata.html.erb]: For this overwrites in `app/views/base` I think you should be able to instead put a different of the template in `app/views/numismatics_issue`

**Comment by christinach** [comment]
Thanks for reviewing @hackmastera . It is still a WIP, we decided with @eliotjordan @jrgriffiniii to add some more layout changes.

File: app/views/base/_form_metadata.html.erb (hunk 1 above)
Comment (christinach) [reply app/views/base/_form_metadata.html.erb]: I will check this. thank you.

**Comment by christinach** [comment]
@eliotjordan @jrgriffiniii if you could review this PR because it starts to get big. I haven't added yet: 1. the toggle sidebar, 2. the folder icon in the monograms panel heading and the sticky functionality for the numismatic buttons at the end for the form. Thanks

File: app/resources/numismatics/issues/numismatics/issue.rb [hunk 2]
```diff
@@ -55,8 +55,6 @@ class Issue < Resource
     attribute :depositor
     attribute :identifier
     attribute :replaces
-    attribute :rights_statement
-    attribute :rights_note, Valkyrie::Types::Set
```
Comment (eliotjordan) [inline app/resources/numismatics/issues/numismatics/issue.rb]: I don't think it's a good idea to remove these attributes completely. Maybe choose an appropriate default and just hide them on the edit page.

File: app/resources/numismatics/coins/numismatics/coin_decorator.rb [hunk 3]
```diff
@@ -11,7 +11,6 @@ class CoinDecorator < Valkyrie::ResourceDecorator
             :analysis,
             :public_note,
             :private_note,
-            :rights_statement,
```
Comment (eliotjordan) [inline app/resources/numismatics/coins/numismatics/coin_decorator.rb]: I think it's fine to display the rights statement. See note on `issue.rb`.

File: app/resources/numismatics/coins/numismatics/coin_change_set.rb [hunk 4]
```diff
@@ -28,7 +28,6 @@ class CoinChangeSet < ChangeSet
     property :replaces, multiple: true, required: false, default: []
     property :depositor, multiple: false, required: false
     property :read_groups, multiple: true, required: false
-    property :rights_statement, multiple: false, required: true, default: RightsStatements.no_known_copyright, type: ::Types::URI
```
Comment (eliotjordan) [inline app/resources/numismatics/coins/numismatics/coin_change_set.rb]: See note on `issue.rb`.

File: app/resources/numismatics/coins/numismatics/coin.rb [hunk 5]
```diff
@@ -35,7 +35,6 @@ class Coin < Resource
     attribute :weight
     attribute :find_number
     attribute :numismatic_collection
-    attribute :rights_statement
```
Comment (eliotjordan) [inline app/resources/numismatics/coins/numismatics/coin.rb]: See note on `issue.rb`.

File: spec/resources/numismatics/coins/numismatics/coin_spec.rb [hunk 6]
```diff
@@ -2,16 +2,12 @@
...
+  let(:coin) { FactoryBot.create_for_repository(:coin, weight: 5, identifier: "ark:/99999/fk4") }
 
   it "has properties" do
     expect(coin.weight).to eq([5])
   end
 
-  it "has rights_statement" do
-    expect(coin.rights_statement).not_to be nil
-  end
-
```
Comment (eliotjordan) [inline spec/resources/numismatics/coins/numismatics/coin_spec.rb]: See note on `issue.rb`.

File: spec/resources/numismatics/issues/numismatics/issues_feature_spec.rb [hunk 7]
```diff
@@ -188,7 +186,7 @@
 
     scenario "viewing a resource" do
       visit solr_document_path numismatic_issue
-      expect(page).to have_css ".attribute.rendered_rights_statement", text: "Copyright Not Evaluated"
+      expect(page).not_to have_css ".attribute.rendered_rights_statement", text: "Copyright Not Evaluated"
```
Comment (eliotjordan) [inline spec/resources/numismatics/issues/numismatics/issues_feature_spec.rb]: See note on `issue.rb`.

**Comment by eliotjordan** [comment]
@christinach Can you say more about the toggle sidebar? I don't remember that discussion very well.

File: app/resources/numismatics/issues/numismatics/issue.rb (hunk 2 above)
Comment (christinach) [reply app/resources/numismatics/issues/numismatics/issue.rb]: So we should keep a default no copyright in the show page and hide them from the edit and the new form. Thanks! 
Screen Shot 2020-02-19 at 4 46 16 PM

**Comment by christinach** [comment]
We mentioned at some point to add a sidebar so that the users can navigate through the page without too much scrolling. @jrgriffiniii suggested to be a toggle sidebar so that they can show/hide it on the side.

**Comment by christinach** [comment]
something like https://codepen.io/ScintillaLuz/pen/HxzED (ignore the red !)

**Comment by christinach** [comment]
closes #3681 , #3655 , #3657 , #3624, #3623

**Pull request merged**